front/.storybook/mockServer.ts

```diff
@@ -50,6 +53,8 @@ const endpoints = {
 
   ...phasesEndpoints,
 
+  ...phasesMiniEndpoints,
```

Comment:
Storybook was broken because I forgot to add this new mock endpoint- not related to other changes